db/migrate/20161205180535_add_load_balancer_id_to_floating_ips.rb

```diff
@@ -0,0 +1,9 @@
+class AddLoadBalancerIdToFloatingIps < ActiveRecord::Migration[5.0]
+  def up
```

Comment:
Note: when you are using change method, column is automatically removed during rollback.
```ruby
def change
  add_column :floating_ips, :load_balancer_id, :bigint
end
```